logger.go

```diff
@@ -44,6 +45,13 @@ func (l LevelLogger) DebugPrintf(format string, a ...interface{}) {
 	}
 }
 
+func (l LevelLogger) TracePrintf(format string, a ...interface{}) {
```

Comment:
How about "TracePrint" and "TracePrintln" ?

Reply:
Added